docs/documentation/automation/question_exec.md

```diff
@@ -6,4 +6,105 @@ sidebar_label: Question Exec
 import useBaseUrl from '@docusaurus/useBaseUrl'; 
 import Highlight from '@theme/Highlight';
 
-# Subtitle {#subtitle}
+<span class="hero__subtitle">
+
+_Customize Survey Questions with Javascript Code_
+
+</span>
+
+## Overview
+Surveys are among Cotalker's main features. Their flexibility and customization ability allow them to adapt to each business's needs.
+
+This spec expands the notion of survey functionality by introducing _Question EXEC_, custom code execution for your survey questions.
+
+:::info
+- Custom code must be written in **Javascript**.
```

Comment:
@guillevalin ¿What version should we say that we support? ECMAScript 2015 ?

![image](https://user-images.githubusercontent.com/3951895/113779731-26b12280-96fc-11eb-9423-4bfd0ba63a48.png)